### ISSUE TITLE
Dynamic font size improvements

### ISSUE THREAD

**Issue by devinreams** [mozilla-lockwise/lockwise-ios#512]
Follow up to #491 #470

- [x] Confused about why icons are behaving in that manner, as well as truncation acting two different ways https://github.com/mozilla-lockbox/lockbox-ios/pull/628
  <img width="376" alt="screen shot 2018-06-27 at 9 37 58 am" src="https://user-images.githubusercontent.com/6164801/41984529-24db75e2-79ee-11e8-99ee-19fd9e39cc71.png">

- [x] "Lock Now" button not getting bigger relative to font size, also losing padding from bottom
  <img width="377" alt="screen shot 2018-06-27 at 9 38 23 am" src="https://user-images.githubusercontent.com/6164801/41984599-45dfe3a4-79ee-11e8-81c1-ac3d01d370ec.png">

- [ ] Why is search truncating so early, relative to the space available?
  <img width="376" alt="screen shot 2018-06-27 at 9 38 36 am" src="https://user-images.githubusercontent.com/6164801/41984667-734a77d2-79ee-11e8-887e-1a738d1c6a16.png">

- [ ] "Learn how to edit this entry" appears to be fixed to the base of the screen, overlapping with the rest of the content on the detail... 
  <img width="393" alt="screen shot 2018-06-27 at 9 38 48 am" src="https://user-images.githubusercontent.com/6164801/41984687-84e5d9fa-79ee-11e8-8098-dc59c8fa1ac5.png">

- [ ] Unlock prompt getting cut off on the bottom
  <img width="390" alt="screen shot 2018-06-27 at 9 44 51 am" src="https://user-images.githubusercontent.com/6164801/41984834-e3e0edbe-79ee-11e8-90da-61c500160b4a.png">

- [ ] I noticed the second bullet point on the onboarding confirmation screen (encryption) was not dynamically sized. It appears to be a text view, not a label like the other two. Should that be swapped out?

**Comment by sigito** [comment]
Overall Apple doesn't scale navigation icons, title, and search bars. Should lockbox also not try to scale those icons?

**Comment by changecourse** [comment]
Agreed @sigito, I think we'd want to lock those icons down so they aren't adversely affecting the layout, and are in-line with Apple's heuristics.
